database/migration/migrate-app_min_version.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE public.app_config
+ADD COLUMN app_min_version text NOT NULL;
```

Comment:
Let's stick with newlines at the EOF for POSIX compliance, in case the sql file gets piped into a script.